docs/sdks/client-sdks/javascript/precomputed-assignments.mdx

```diff
@@ -63,3 +63,139 @@ After the precomputed Eppo client is initialized, the client instance can be acc
 const client = EppoSdk.getPrecomputedInstance();
 const variant = client.getStringAssignment(flagKey, 'default-value');
 ```
+
+## Offline Precomputed Assignments
+
+### Overview
+
+The Eppo Node Server SDK can be used to generate precomputed assignments on the server and bootstrap the client-side SDK using `offlinePrecomputedInit`. This approach allows you to initialize the client without making remote calls from the browser.
+
+These SDK methods are available in Eppo's Javascript SDK version `3.9.4` and above.
+
+### Initialize offline precomputed client
+
+First, generate the precomputed configuration on your backend using Eppo's Node Server SDK:
+
+```javascript
+import * as EppoSDK from '@eppo/node-server-sdk';
+
+await EppoSdk.init({
+  apiKey: 'YOUR_SDK_KEY',
+  assignmentLogger: { logAssignment() {} }, // we will log in the client, not here
```

Comment:
side note: we need to edit our types to make this nullable so we don't have to pass in empty functions like this. It will work as we do a truthiness check, we just need to edit our type.